### PR TITLE
ASUIS-346: Fix User form grammar.

### PR DESCRIPTION
**Pull request by georgebaev**
@mlsamuelson this fixes User form's grammar.

Ref.: https://asudev.jira.com/browse/ASUIS-346

### CODE REVIEW

File: web/modules/custom/webdir_profile/js/webdir-profile.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * @file
+ * Web Directory Profile behaviors.
+ */
+
+(function ($, Drupal) {
+
+  'use strict';
+
+  /**
+   * Profile related JavaScript functioality.
```
Comment (mlsamuelson) [inline web/modules/custom/webdir_profile/js/webdir-profile.js:11]: Typo: `functioality` should be `functionality`.

File: config/field.field.user.user.user_picture.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ field_name: user_picture
...
+description: "<p>Format and Size: You may upload a JPEG, JPG, GIF, or PNG image to use as your profile image. It is recommended the image not be smaller than 150 pixels by 150 pixels and not larger than 800 pixels by 800 pixels. Also note, if you attempt to upload an image with excessive file size, the upload may fail. </p>\r\n<p>Please use common sense when uploading a profile image. If we receive complaints about the content in your directory profile, including the profile image, ASU may exercise its right to remove the image without warning and suspend your ability to customize your profile. </p>\r\n<br /><strong>Photo file requirements</strong>"
 required: false
```
Comment (mlsamuelson) [inline config/field.field.user.user.user_picture.yml:18]: Comma added in this line should be removed - no comma after GIF in: `JPEG, JPG, GIF or PNG`
This is in compliance with Brand Guide Standards: https://brandguide.asu.edu/brand-elements/writing-content/punctuation#:~:text=ASU%20style%20does%20not%20recognize,is%20red%2C%20white%20and%20blue.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

File: config/field.field.user.user.user_picture.yml (hunk 2 above)
Comment (georgebaev) [reply config/field.field.user.user.user_picture.yml:18]: This is fixed

File: web/modules/custom/webdir_profile/js/webdir-profile.js (hunk 1 above)
Comment (georgebaev) [reply web/modules/custom/webdir_profile/js/webdir-profile.js]: This is fixed.

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**